ATM/ATM.py

```diff
@@ -0,0 +1,164 @@
+import time
+
+def set_pin():
+    user_input = None
+    while not pin_input_well_formed_check(user_input):
+        user_input = input("Please choose your four digit PIN. ")
+    return user_input
+
+
+def pin_input_well_formed_check(pin_input_to_be_checked):
```

Comment:
although what you did works, these kind of test are usually done with regular expressions ( or REGEX as it is usually called). please refactor your code to work with REGEX.
this site: https://regex101.com could be very useful to do that.

Reply:
done